ClosedXML/Excel/XLWorksheet.cs

```diff
@@ -584,8 +584,9 @@ public IXLWorksheet CopyTo(XLWorkbook workbook, String newSheetName, Int32 posit
             foreach (var picture in Pictures)
             {
                 var newPic = targetSheet.AddPicture(picture.ImageStream, picture.Format, picture.Name)
-                    .WithPlacement(picture.Placement)
-                    .WithSize(picture.Width, picture.Height);
+                    .WithPlacement(XLPicturePlacement.FreeFloating)
```

Comment:
Isn't it sufficient to have only 2 lines? With Size first, and placement second?

Reply:
Sorry, I got busy and missed your comments.  The problem is that there is an image with the `MoveAndSize` placement type saved in that excel file.  Trying to set the size in `WithSize` throws exceptions for images with this anchor type. See here: https://github.com/ClosedXML/ClosedXML/blob/230236e6ac43ff27c9947c966dd9cd1bc23ee06c/ClosedXML/Excel/Drawings/XLPicture.cs#L223

This PR just sets the placement first to something less restrictive on sizing and then resets it afterwards to the original anchor type.  The other alternative I see would be to remove the lines that throw exceptions in XLPicture.  You might have a better idea of how to fix it that I'm not aware of though.

Reply:
Ok, I'm happy with this.